rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/identifier.hpp

```diff
@@ -18,7 +18,7 @@
 namespace rosidl_typesupport_introspection_cpp
 {
 
-static const char * typesupport_introspection_identifier = "introspection";
+static const char * typesupport_introspection_identifier;
```

Comment:
Because you usually don't use `static` symbols from other libraries. Maybe it is necessary for how you are exporting the symbol from `identifier.cpp` to the libraries and executables which use the symbol, but I don't know. It seems to me that `const` is sufficient since the only thing `static` adds to this is setting the linkage to internal, which doesn't apply since it is a declaration and not an initialization. At any rate I think the question should not by "why shouldn't it be `static`" but rather "why should it be `static`"?

If it works as-is that's fine, I'd just like to understand the reason for it being that way.